### DISCUSSION TITLE
Discord Server

### DISCUSSION THREAD

**Discussion by coopbri** [foundry-rs/foundry#4708] [General]
In some of the GH issues, I see mention of a Discord server. Is there a public server available for the Foundry ecosystem?

**Comment by Azleal** [comment]
is there a official dc server?

**Comment by BrokedTV** [comment]
It would be nice, it's very easy to ask stuff and search for keywords

**Comment by CarlosSchell** [comment]
╔═╗ ╔═╗ ╦ ╦ ╔╗╔ ╔╦╗ ╦═╗ ╦ ╦         Portable and modular toolkit
 ╠╣  ║ ║ ║ ║ ║║║  ║║ ╠╦╝ ╚╦╝    for Ethereum Application Development
 ╚   ╚═╝ ╚═╝ ╝╚╝ ═╩╝ ╩╚═  ╩                 written in Rust.

.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx

Repo       : https://github.com/foundry-rs/
Book       : https://book.getfoundry.sh/
Chat       : https://t.me/foundry_rs/
Support    : https://t.me/foundry_support/
Contribute : https://github.com/orgs/foundry-rs/projects/2/

.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx
